client/checkout/utils/fingerprint.js

```diff
@@ -27,14 +27,14 @@ export const getFingerprint = async () => {
 /**
  * Appends a hidden input with the user fingerprint to the checkout form.
  *
- * @param {Object} form        The jQuery Checkout form object.
+ * @param {Object} $form        The jQuery Checkout form object.
  * @param {string} fingerprint User fingerprint.
  */
-export const appendFingerprintInputToForm = ( form, fingerprint = '' ) => {
+export const appendFingerprintInputToForm = ( $form, fingerprint = '' ) => {
```

Comment:
Ensuring consistent naming convention - prepending the `$` in the name of the `form` argument, since it's a jQuery object